validate.lic

```diff
@@ -512,6 +518,7 @@ class DRYamlValidator
     @valid_weapon_skills = @valid_ranged_skills + @valid_melee_skills + ['Targeted Magic']
     @valid_summon_skills = @valid_thrown_skills + @valid_melee_skills - ['Offhand Weapon', 'Brawling']
     @valid_research_skills = ['Arcana', 'Life Magic', 'Holy Magic', 'Lunar Magic', 'Elemental Magic', 'Arcane Magic', 'Attunement', 'Warding', 'Augmentation', 'Utility']
+    @valid_defensive_skills = ['Shield Usage', 'Parry Ability', 'Evasion']
```

Comment:
It's not part of this PR, but you've made me think that another good validator would be to check the defense skills in the `stances` setting. Just a thought for the future...